src/app/features/psbt-signer/components/psbt-inputs-outputs-totals/psbt-inputs-outputs-totals.tsx

```diff
@@ -34,7 +32,9 @@ export function PsbtInputsOutputsTotals() {
           <PsbtAddressTransferTotals showNativeSegwitTotal={isNativeSegwitTotalGreaterThanZero} />
         </Box>
       ) : null}
-      {showDivider ? <Hr backgroundColor={color('border')} height="3px" /> : null}
+      {showDivider ? (
+        <styled.hr border="1px solid" borderColor="accent.border-default !important" />
```

Comment:
Curious why the `!important` is necessary. Known issue with panda?

Reply:
I really don't know here. I can't seem to override the borderColor unless I do this. I've had to do it in several places but haven't yet figured out a better solution. 🤔